### ISSUE TITLE
Fails with "You are trying to build OCaml native code file ... Include the following definition in your OMakefile       if you really want to build this file. NATIVE_ENABLED = true" regardless of NATIVE_ENABLED = true present in OMakefile

### ISSUE THREAD

**Issue by yurivict** [ocaml-omake/omake#124]
On FreeBSD this project https://github.com/teyjus/teyjus fails regardless of ```NATIVE_ENABLED = true``` being present in the ```OMakefile```:

```
*** omake: failed (0.17 sec, 0/0 scans, 0/0 rules, 88/525 digests)
*** omake error:
   File /usr/local/lib/omake/build/OCaml.om: line 493, characters 12-24
   Uncaught Exception:
      You are trying to build OCaml native code file: lplex.o
      However, the NATIVE_ENABLED flag is not set.
      Include the following definition in your OMakefile
      if you really want to build this file.
             NATIVE_ENABLED = true
*** Error code 123
```

**Comment by cspiel** [comment]
**ocamlopt** found?

**Comment by yurivict** [comment]
> ocamlopt found?

It wasn't found. When it is present the error is gone.

This makes it the case of misleading error message. It should have said "ocamlopt not found" instead of "the NATIVE_ENABLED flag is not set".

**Comment by cspiel** [comment]
If your project _requires_ the native-code OCaml compiler it ought to test for it
and not carry on blindly.

On the OMake side we know which compilers are available at the point where
the exception is raised.  So the message of the exception can be made much more
precise.  I have opened pull request #126; its class structure is still unsatisfying
but it helps with your problem.

**Comment by cspiel** [comment]
Merged #126
